skutil/h2o/java_templating/tests/test_templater.py

```diff
@@ -0,0 +1,38 @@
+from skutil.h2o.java_templating.templater import Templater
+import os
+import sys
+
+
+def test_templater():
+
+    templates_dir = os.path.dirname(os.path.realpath(__file__)) + "/../../templates/"
+    template_filename = "main_class.java"
+
+    # attempt to load template file
+    try:
+        with open(templates_dir + template_filename, 'r') as myfile:
+            template_string = myfile.read()
+    except IOError as io_err:
+        print("\n\n" + io_err.strerror + ": " + template_filename + "\n")
+        sys.exit(-1)
+
+    # replace values within template with values in kwargs
+    kwargs = {"model_class_name": "gbm_test",
+              "fields": """
+        RowData row = new RowData();
```

Comment:
Fair enough! I am interested in the alternative. I do feel this can be a bit cobbled together especially when dealing with indent levels and long strings. Example of the alternative?